src/components/CytoscapeGraph/Layout/GroupCompoundLayout.ts

```diff
@@ -0,0 +1,174 @@
+const NAMESPACE_KEY = '_group_compound_layout';
```

Comment:
This file would benefit from some comments.  It will be hard for anyone looking at this code later to make changes, it's not simple.  Add some high-level approach and some lower level details in tricky spots.  I know what you are tying to do and I'm only partially grokking it.  Maybe explain that this is a new "synthetic" layout, how the bounding box is calculated, etc..  As for the bounding box, I don't really understand, it is creating a single large node to replace the member nodes?  How is that bounding box sized, is it somehow based on the original node sizes?

Reply:
> As for the bounding box, I don't really understand, it is creating a single large node to replace the member nodes? How is that bounding box sized, is it somehow based on the original node sizes?

Is using the parent as a large single node, using the size of the children.

I'll add more comments into the file to make it easier to follow up